core/app/models/spree/product/scopes.rb

```diff
@@ -166,7 +164,7 @@ def self.property_conditions(property)
     # :order => 'COALESCE(cnt, 0) DESC'
     add_search_scope :descend_by_popularity do
       joins(:master).
-      order(%Q{
+        order(%Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.